assets/src/edit-story/components/checklist/checks/elementLinkTappableRegionTooSmall.js

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import { sprintf, __ } from '@web-stories-wp/i18n';
+
+/**
+ * Internal dependencies
+ */
+import { Text, THEME_CONSTANTS } from '../../../../design-system';
+import { useStory } from '../../../app';
+import { ChecklistCard } from '../../checklistCard';
+import { filterStoryElements } from '../utils';
+
+const LINK_TAPPABLE_REGION_MIN_WIDTH = 48;
+const LINK_TAPPABLE_REGION_MIN_HEIGHT = 48;
+
+export function elementLinkTappableRegionTooSmall(element) {
+  if (
+    !['text', 'image', 'shape', 'gif', 'video'].includes(element.type) ||
+    !element.link?.url?.length
+  ) {
+    return false;
+  }
+
+  return (
+    element.width < LINK_TAPPABLE_REGION_MIN_WIDTH ||
+    element.height < LINK_TAPPABLE_REGION_MIN_HEIGHT
+  );
+}
+
+const ElementLinkTappableRegionTooSmall = () => {
+  const story = useStory(({ state }) => state);
+  const elements = filterStoryElements(
+    story,
+    elementLinkTappableRegionTooSmall
+  );
+  return (
+    elements.length > 0 && (
+      <ChecklistCard
+        title={sprintf(
+          /* translators: %s: minimum tappable region size width x minimum tappable region size height. */
```

Comment:
I'm good with this here for now, but once all of these checks are merged I think I'll do a separate PR to pull the new constants into this updated directory and import copy and constants throughout.  As far as things go, for how many different PRs there've been to build this quickly it's overall not the worst thing that the constants are the thing that got lost!